src/main/java/io/cdap/delta/bigquery/BigQueryTarget.java

```diff
@@ -147,22 +159,23 @@ public void initialize(DeltaTargetContext context) throws Exception {
   @Override
   public EventConsumer createConsumer(DeltaTargetContext context) throws IOException {
     Credentials credentials = conf.getCredentials();
-    String project = conf.getProject();
+    if (conf.getProject() == null) {
+      throw new RuntimeException("Project id is not Present");
```

Comment:
similar : move this inside the `getProject()` , will reduce code repetition.

Reply:
Updated and Removed